docker-compose.yml

```diff
@@ -27,7 +27,9 @@ services:
     build:
       context: ./vivo
     volumes:
-      - ./example-config:/usr/local/vivo/home/config
+      - ./vivo/applicationSetup.n3:/usr/local/VIVO/home/config/applicationSetup.n3
+      - ./vivo/log4j.properties:/usr/local/tomcat/webapps/VIVO/WEB-INF/classes
```

Comment:
The VIVO application is actually being deployed at:
`/usr/local/tomcat/webapps/vivo`
as opposed to:
`/usr/local/tomcat/webapps/VIVO`

Additionally, this command appears to be creating a file named `classes` with the contents of `log4j.properties`.

Reply:
This conversation highlights a bug in the ./vivo/Dockerfile
I stole this filepath from line 35 of ./vivo/Dockerfile, but line 35 can't be correct.  As you're saying, it does two things wrong: it mounts a file over a directory, and it's the wrong directory.
If that's true, line 35 of ./vivo/Dockerfile should read:
`COPY ./log4j.properties /usr/local/tomcat/webapps/vivo/WEB-INF/classes/log4j.properties` or something
and the docker-compose.yml line 31 should read:
`- ./vivo/log4j.properties:/usr/local/tomcat/webapps/vivo/WEB-INF/classes/log4j.properties` or something

Reply:
(The comments you previously made about a different part of the PR, Github hid the comments after I undid those lines like you recommended.)